cranelift/codegen/src/isa/aarch64/inst.isle

```diff
@@ -3245,7 +3245,7 @@
       (fpu_to_int op src))
 (rule (fpu_to_int_cvt_sat op src $false (fits_in_16 out_ty))
       (let ((result Reg (fpu_to_int op src))
-            (max Reg (imm out_ty (ImmExtend.Zero) -1)))
+            (max Reg (imm out_ty (ImmExtend.Zero) (u64_not 0))))
```

Comment:
Here, can we spell out 0xffff_ffff_ffff_ffff? It's kind of awkward, but otherwise we have an extra call to an external constructor that may or may not be inlined + constant-propagated.